### DISCUSSION TITLE
Python connect Forticlient vpn fails in script, works on commandline

### DISCUSSION THREAD

**Discussion by cadvantages** [community/community#36479] [Other Feature Feedback, Questions, & Ideas]
Hi Guys,
i am new to the forum, average Python developer. I need ur advise on connection to Forticlient vpn in python script. Connection works fine from Python command line but fails in my  script, hangs does not open the forticlient window 
script options used
os.system('C:\\Program Files\\Fortinet\\FortiClient\\Fortinet.exe')

subprocess.Popen("C:\\Program Files\\Fortinet\\FortiClient\\FortiClient.exe", shell=True,stdout=subprocess.PIPE,stderr=subprocess.PIPE).communicate()[0]

Got  error " "Uncaught error in main process read ENOTCONN"  for some test run .
Any assist will be appreciated. 
Thnx.

**Comment by ossanna16** [comment]
Hi @cadvantages ,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Python. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
